backend/src/apiserver/server/visualization_server.go

```diff
@@ -115,13 +113,17 @@ func (s *VisualizationServer) getVisualizationServiceURL(request *go_client.Crea
 	return s.serviceURL
 }
 
-func isVisualizationServiceAlive(serviceURL string) bool {
+func isVisualizationServiceAlive(serviceURL string) error {
 	resp, err := http.Get(serviceURL)
+
 	if err != nil {
-		glog.Error("Unable to verify visualization service is alive!", err)
-		return false
+		glog.Error(fmt.Sprintf("Unable to verify visualization service aliveness by sending request to %s and get error: %s", serviceURL, err.Error()))
```

Comment:
Also, you can probably reduce duplication by instantiating the error object first then glog the error object.

Reply:
Fixed